### PR TITLE
Feature/cli doc

### PR DESCRIPTION
**Pull request by james-haytko-sp**

### CODE REVIEW

File: products/idn/tools/cli/index.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+[configure PAT](./assets/img/vhs/configure/configure-pat.gif)
+
+If you are using an **OAuth client**, run this command: 
+
+```shell
+sail configure oauth
+```
+
+[configure OAuth](./assets/img/vhs/configure/configure-oauth.gif)
```
Comment (luke-hagar-sp) [inline products/idn/tools/cli/index.md:119]: OAuth is not currently supported

File: products/idn/tools/cli/index.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+```powershell
+New-Item -ItemType Directory -Path 'C:\Users\<username>\.sailpoint'
+New-Item -ItemType File -Path 'C:\Users\<username>\.sailpoint\config.yaml'
+```
+These commands will create a `config.yaml` file you can use to set your environment variables:
+
+The `config.yaml` file must contain the following information:
+
+```yaml
```
Comment (luke-hagar-sp) [inline products/idn/tools/cli/index.md:143]: This configuration file structure has been updated on the CLI to reflect the new one

File: products/idn/tools/cli/index.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+You can have both authentication methods configured at once, but only one can be active at a time. 
+
+#### Environment variable configuration 
+
+You can also store your configuration in environment variables. 
+On **Linux/Mac**, open your terminal app and run these commands to export the following environment variables:
+
+```shell
+export SAIL_BASEURL=https://{tenant}.api.identitynow.com
```
Comment (luke-hagar-sp) [inline products/idn/tools/cli/index.md:179]: The ENV VARS for each of these sections have changed to snake case, examples should be on the main readme of the CLI

File: products/idn/tools/cli/index.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+```powershell
+[System.Environment]::SetEnvironmentVariable('SAIL_BASEURL','https://{tenant}.api.identitynow.com')
+```
+
+## Usage
+
+Run the `sail` command for an overview of the available commands and flags. You can use the `-h` flag with any available command to see additional available options for each command: 
+
+The following commands are available.:
```
Comment (luke-hagar-sp) [inline products/idn/tools/cli/index.md:206]: We have more commands, I'm currently in the process of creating these nested files for them, whats the best way for me to get that content to you?

**Review by luke-hagar-sp — CHANGES_REQUESTED** [review]
Hey James!

This looks good, I have additional documentation for the Other commands I'm working on now, whats the best way for me to get that content to you?

Additionally I had some comments on some small items, I would suggest looking at the CLI Readme now that the most recent big PR is merged.

Thank you for all you do!

File: products/idn/tools/cli/index.md (hunk 1 above)
Comment (james-haytko-sp) [reply products/idn/tools/cli/index.md:119]: Oh, that's right. I'll copy that somewhere else for now.

Comment (luke-hagar-sp) [reply products/idn/tools/cli/index.md:119]: I actually just commented it out on the current CLI Readme

**Comment by luke-hagar-sp** [comment]
It looks like the PR build is failing too, it says it found broken links?

File: products/idn/tools/cli/index.md (hunk 4 above)
Comment (james-haytko-sp) [reply products/idn/tools/cli/index.md:206]: Okay nice, you can just message me and I'll add it in, or keep updating the readme and I'll be better about checking it, or you can update my branch directly and throw them in there, and I'll edit them. Whatever is easiest for you works for me.

File: products/idn/tools/cli/index.md (hunk 3 above)
Comment (james-haytko-sp) [reply products/idn/tools/cli/index.md]: Okay I updated these to reflect the updates in the readme.

File: products/idn/tools/cli/index.md (hunk 2 above)
Comment (james-haytko-sp) [reply products/idn/tools/cli/index.md:135]: Okay I copied in the new config.yaml file from the updated readme.

File: products/idn/tools/cli/connectors.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+- [Init project](#init-project)
+- [Create connector](#create-connector)
+- [Upload connector](#upload-connector)
+- [Invoke command](#invoke-command)
+- [List connectors](#list-connectors)
+- [Update connector](#update-connector)
+- [Delete connector](#delete-connector)
+- [Manage tags](#manage-tags)
+- [Get logs](#get-logs)
```
Comment (philip-ellis-sp) [inline products/idn/tools/cli/connectors.md:32]: We are missing the sail conn validate command here. The validate command runs integration tests against your connector to ensure all the basic functionality works. You can pass in a -r to run it as read only or do a full suite of read write tests.

**Review by philip-ellis-sp — CHANGES_REQUESTED** [review]

**Comment by jordan-violet-sp** [comment]
We should change the name of the document "SET" to something more human readable for what that function does, e.g. "Configuration" or something along those lines.

**Pull request closed without merging**